scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -714,21 +716,6 @@ checkPeers() {
   fi
 }
 
-checkNodeVersion() {
-  version=$("${CNODEBIN}" version)
-  node_version=$(grep "cardano-node" <<< "${version}" | cut -d ' ' -f2)
-  node_rev=$(grep "git rev" <<< "${version}" | cut -d ' ' -f3 | cut -c1-8)
-
-  if [[ ${node_version} != "${running_node_version}" || ${node_rev} != "${running_node_rev}" ]]; then
-    clrScreen
-    printf "\n ${style_status_3}Node version mismatch${NC} - running version doesn't match found binary!"
-    printf "\n\n Forgot to restart node after upgrade?"
-    printf "\n\n Deployed version : ${node_version} (${node_rev}) => ${CNODEBIN}"
-    printf "\n Running version  : ${running_node_version} (${running_node_rev})\n"
-    waitToProceed && clrScreen
-  fi
-}
-
```

Comment:
I think this is an artifact from the source branch starting from alpha before recent commits were merged. I would have thought your rebase off upstream/alpha would have dropped this change, bit it appears to be stored in the commit instead of cleared up by the rebase operation.